adobe/react-spectrum
# Making TableView columns announce sort info and sort order changes

Closes #2063 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Go to tableview--async-loading story with VoiceOver/JAWS/NVDA
2. Navigate to the column headers and make sure their sortablity is announced
3. Change the sort order of the column and make sure it is announced


Known issues:
- Announcements of table sort order changes are quite flakey on iPhone/iPad VoiceOver. Either they cut out or don't announce at all.
- iOS VoiceOver doesn't announce the `aria-describedby` content of the table columns at all, happens with native `th` elements as well (https://xyi4e.csb.app/)
- You must tap on/navigate off of and back on the column header in iOS VoiceOver to hear the aria-sort of the current column  
## 🧢 Your Project:

RSP
